pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
 markers =
   working: test should work
-  integrated: this is an integrated test
+  integrated: this is an integration test
+  integratedx: this integration test redundantly tests functionality covered in unit tests
```

Comment:
Is this the difference? `integratedx` is redundant?

Reply:
I fixed the doc in pytest.ini to hopefully be more clear:
```
  integrated: an integration test
  integratedx: an excludable integration test, redundantly testing functionality also covered by a unit test
```
I also fixed the make targets in the `Makefile` to hopefully be more clear:
```
test:  # runs default tests, which are the unit tests
	make test-units

test-all:
	pytest -vv

test-units:  # runs unit tests (and integration tests not backed by a unit test)
	pytest -vv -m "not integratedx"

test-integrations:  # runs integration tests
	pytest -vv -m "integrated or integratedx"
```